Fix a warning in fsl-rngc.c

Interface has been changed and driver needs a minor update.

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 drivers/char/hw_random/fsl-rngc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.26/drivers/char/hw_random/fsl-rngc.c
===================================================================
--- linux-2.6.26.orig/drivers/char/hw_random/fsl-rngc.c	2009-04-24 21:44:01.000000000 +0200
+++ linux-2.6.26/drivers/char/hw_random/fsl-rngc.c	2009-04-24 21:44:33.000000000 +0200
@@ -124,7 +124,7 @@ static struct platform_device *rng_dev;
 
 int irq_rng;
 
-static int fsl_rngc_data_present(struct hwrng *rng)
+static int fsl_rngc_data_present(struct hwrng *rng, int wait)
 {
 	int level;
 	u32 rngc_base = (u32) rng->priv;
